src/main/java/Parser.java

```diff
@@ -0,0 +1,180 @@
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+    public Command firstParse(String input) {
```

Comment:
I understand this first parses the input and gets the command, maybe parseCommand would be clearer 😄